src/Quill.php

```diff
@@ -158,91 +147,80 @@ class Quill extends InputWidget
      * @since 2.0.0
      */
     public $highlightStyle = 'default.min.css';
```

Comment:
Previously for local version only default css was available and the minimized version was added manually. Right now since all styles are available we can easily allow to use them on local asset but there is one problem - CDN serves only minimized styles and in the NPM package there are only non-minimized ones. I suggest changing this property to take only style name and to add `.css` in local asset and `.min.css` in CDN asset.